source/includes/_10-one-time-sync-company-custom-check.html.md.erb

```diff
@@ -0,0 +1,547 @@
+# One-time Sync Company Custom 
+
+This section lists all endpoints that must be implemented for a one-time synchronous company custom check integration.
+
+One-time synchronous company custom checks are custom checks with the following behaviour:
+
+    * The check result will be returned directly upon request; the response will not be sent until the check is finished (synchronous)
+    
+    * The check operates at a point-in-time: there is no ongoing monitoring of the check result (one-time)
+
+
+## Get Check Configuration (One-time Sync Custom Company)
+
+<%= partial("includes/partials/config.md.erb", :locals => { :name => "sync_company_custom" }) %>
+
+This endpoint specifies the configuration used by PassFort for performing
+one-time synchronous custom checks using your integration.
+
+### HTTP Request
+
+`GET https://my-integration.example.com/config`
+
+### Response fields
+
+Most configuration fields are already discussed in the [configuration
+section](#configuration). Check specific options will be discussed here.
```

Comment:
Suggest reworking second sentence: "The following options are specific to the Company custom check."